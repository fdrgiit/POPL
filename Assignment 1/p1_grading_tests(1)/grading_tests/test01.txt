
111    +-+*
111...111*****+++++ 111.abc
111.110 011

111
Z123.12RE01
STRING.00INT
STRING.0INT

