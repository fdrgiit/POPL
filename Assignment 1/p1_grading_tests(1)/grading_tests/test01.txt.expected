2 34 111 
2 13 +
2 14 -
2 13 +
2 17 *
3 34 111 
3 33 .
3 33 .
3 33 .
3 34 111 
3 17 *
3 17 *
3 17 *
3 17 *
3 17 *
3 13 +
3 13 +
3 13 +
3 13 +
3 13 +
3 34 111 
3 33 .
3 37 abc 
4 34 111 
4 33 .
4 34 110 
4 34 0 
4 34 11 
6 34 111 
7 37 Z123 
7 33 .
7 34 12 
7 37 RE01 
8 12 STRING
8 33 .
8 34 0 
8 34 0 
8 10 INT
9 35 STRING.0INT 
